client/web/src/enterprise/batches/detail/changesets/BatchChangeChangesets.tsx

```diff
@@ -157,6 +157,7 @@ const BatchChangeChangesetsImpl: React.FunctionComponent<Props> = ({
             ...queryArguments,
             first: BATCH_COUNT,
             after: null,
+            checkOpenOrDraft: false
```

Comment:
I would default to null for this.

```suggestion
            checkOpenOrDraft: null
```

Reply:
I would assume since the field is typed as boolean, it's probably best to pass in a value of `false` since it's of the actual type `Boolean`, especially since we're basically doing the truthy check on the backend so this will never really become an issue.

What do you think? @eseliger

Reply:
It is not typed as `boolean`, it is `boolean | null` (`Boolean` vs `Boolean!` in GQL).
I want to indicate here that we don't care about this flag in this instance. `null` vs `false` ought to have different meanings, otherwise we should make it non-optional (but that's a breaking change so let's not do that).

Reply:
That's true. That clarifies. Thanks.